test/java/seedu/addressbook/logic/LogicTest.java

```diff
@@ -13,6 +13,7 @@
 import seedu.addressbook.data.tag.Tag;
 import seedu.addressbook.data.tag.UniqueTagList;
 import seedu.addressbook.storage.StorageFile;
+import seedu.addressbook.parser.Parser;
 
 import java.util.*;
```

Comment:
It is a recommended practice to import only necessary classes instead of whole `util` package.